wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -30,23 +31,7 @@
 public class JobsUtilTest {
 
   @Test
-  public void testEnvVarResolution() throws IOException {
```

Comment:
May still want to verify that env vars are resolved properly?

Reply:
Good point , i have added back earlier code which used to resolve environment variable. In my next PR, i'm going to modify resolving environmental variables using apache commons configuration.